doc/build-osx.md

```diff
@@ -96,11 +96,13 @@ Skip if you don't intend to use the GUI.
 brew install qt@5
 ```
 
-Ensure that the `qt@5` package is installed, not the `qt` package.
-If 'qt' is installed, the build process will fail.
-if installed, remove the `qt` package with the following command:
+Note: Ensure that _only_ `qt@5` is installed:
+> If the `qt` package, which provides Qt 6, is installed alongside `qt@5`, configuring will find Qt 5, but compiling will fail because some headers are incorrectly imported from Qt 6 locations.
```

Comment:
Please unquote this
```suggestion
If the `qt` package, which provides Qt 6, is installed alongside `qt@5`, configuring will find Qt 5, but compiling will fail because some headers are incorrectly imported from Qt 6 locations.
```

Reply:
It's sufficient to say that you can't build if qt6 is installed, we don't need to go into any technical detail. The only information needed is that `qt@5` is the proper package and not `qt6`. I'd still prefer something like this:
```suggestion
If you've previously grabbed the `qt` package, which installs Qt 6, you will not be able to compile.
```

Reply:
@jarolrod I have removed the quote formatting as suggested.

Can you please explain why there is no need to go into rationale that you cannot have both Qt 5 and Qt 6 installed simultaneously?  After-all Qt 6 is becoming quite common now, and it quite reasonable that some developers will think: "Hey, why not? I need Qt 6 for this other project I'm working on.".